modules/serverless-broker-types.adoc

```diff
@@ -1,27 +1,24 @@
 // Module included in the following assemblies:
 //
-// * /serverless/develop/serverless-using-brokers.adoc
+// * /serverless/discover/serverless-brokers.adoc
 
 :_content-type: CONCEPT
 [id="serverless-broker-types_{context}"]
 = Broker types
 
-There are multiple broker implementations available for use with {ServerlessProductName}, each of which have different event delivery guarantees and use different underlying technologies. You can choose the broker implementation when creating a broker by specifying a broker class, otherwise the default broker class is used. The default broker class can be configured by cluster administrators.
-// TO DO: Need to add docs about setting default broker class.
+You can choose the broker implementation when creating a broker by specifying a broker class, otherwise the default broker class is used. The default broker class can be configured by cluster administrators.
```

Comment:
The "class" on its own is not really enough, sometimes.

For instance, on the `Kafka` class, for the Knative Kakfa Broker, we also set `config-br-defaults` config map, like:

```yaml
apiVersion: operator.knative.dev/v1alpha1
kind: KnativeEventing
metadata:
  name: knative-eventing
  namespace: knative-eventing
spec:
  defaultBrokerClass: Kafka
  config:
    config-br-defaults:
      default-br-config: |
        clusterDefault:
          apiVersion: v1
          kind: ConfigMap
          name: kafka-broker-config
          namespace: knative-eventing
```

and the same is true for the "MT Channel Broker", which by default points to the InMemChannel, and comes with a different ConfigMap for that setup, by default:
https://github.com/knative/eventing/blob/main/config/core/configmaps/default-broker.yaml#L26-L32


I know it's hard to explain. 

Good news is that with Kafka Broker things are much easier than with the "MT Channel Broker" + some channel implementation.

Reply:
Sorry, that I just notice this on the sentence, that it may need a bit more

Reply:
ACK, there are other related Jiras to get into the specific configs though for different broker types:

- https://issues.redhat.com/browse/SRVKE-1271
- https://issues.redhat.com/browse/SRVKE-1140
- https://issues.redhat.com/browse/SRVKE-528
- https://issues.redhat.com/browse/SRVKE-528

I will be working on these next, but I think it's too muddy to try to cover all of that in this one PR, and will make reviewing this PR difficult for docs peer reviews. For now I will update this, but I will keep it as a generic statement to say that you can configure it by doing X and Y, I will not cover the full configuration in this PR since it needs its own procedure module etc.

Reply:
> I will be working on these next

perfect and works for me too :smile: